blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -140,12 +142,24 @@ export const actionHandlers = {
         steps[payload.nodesBaseUrl] = payload;
         return state.set('steps', steps);
     },
+    [ACTION_TYPES.REMOVE_STEP](state, { payload }): State {
```

Comment:
Well that is still in my head from my last job. We used it always as "envelope" for the data. This way we had only few attributes in our commands/events, however there is no reason to keep doing it I guess.

I took care of it.